src/main/java/pl/sda/library/domain/model/exception/AuthorAlreadyExistException.java

```diff
@@ -0,0 +1,7 @@
+package pl.sda.library.domain.model.exception;
+
+public class AuthorAlreadyExistException extends ObjectAlreadyExistException {
+    public AuthorAlreadyExistException() {
+        super("Taki autor już istnieje w bazie danych");
```

Comment:
nie piszemy kodu po polsku

Reply:
to jest message do wyświetlenia użytkownikowi. Nie powinien być po polsku?